lua/neorg/modules/core/integrations/telescope/module.lua

```diff
@@ -34,18 +51,58 @@ module.load = function()
 end
 
 module.public = {
-    find_linkable = require("telescope._extensions.neorg.find_linkable"),
-    find_norg_files = require("telescope._extensions.neorg.find_norg_files"),
-    insert_link = require("telescope._extensions.neorg.insert_link"),
-    insert_file_link = require("telescope._extensions.neorg.insert_file_link"),
-    search_headings = require("telescope._extensions.neorg.search_headings"),
-    find_project_tasks = require("telescope._extensions.neorg.find_project_tasks"),
-    find_context_tasks = require("telescope._extensions.neorg.find_context_tasks"),
-    find_aof_tasks = require("telescope._extensions.neorg.find_aof_tasks"),
-    find_aof_project_tasks = require("telescope._extensions.neorg.find_aof_project_tasks"),
-    switch_workspace = require("telescope._extensions.neorg.switch_workspace"),
-    find_backlinks = require("telescope._extensions.neorg.backlinks.file_backlinks"),
-    find_header_backlinks = require("telescope._extensions.neorg.backlinks.header_backlinks"),
+    find_linkable = function(opts)
+        local func = require("telescope._extensions.neorg.find_linkable")(module.config.public.find_linkable)
```

Comment:
why an extra variable and not just
`return require("telescope._extensions.neorg.find_linkable")(module.config.public.find_linkable)(opts)`
ofc same for all the pickers
didn't test but if my lua knowledge isn't wrong this should also work